### PR TITLE
unit test

### PR DESCRIPTION
**Pull request by jburel**
Enable test_gateway and metadata
Fix issue with Python3

### CODE REVIEW

**Comment by jburel** [comment]
Closing this PR
commits moved to https://github.com/ome/omero-py/pull/59

**Comment by jburel** [comment]
Without fa75687
I can see error like
``` Error importing plugin "omero.gateway.pytest_fixtures": cannot import name 'StringTypes'```
when I run 
```
./setup.py test -t test/integration/gatewaytest/test_rdefs.py -k testDefault

```

File: tox.ini [hunk 1]
```diff
@@ -44,9 +45,8 @@ commands =
         test/unit/test_rois.py \
         test/unit/test_tempfiles.py \
         test/unit/test_util.py \
+        test/unit/test_gateway.py \
+        test/unit/test_metadata_utils.py \
```
Comment (manics) [inline tox.ini]: Minor nitpick: List these in alphabetical order?

**Comment by joshmoore** [comment]
Round of merging today with all 3 CI's running even if not (yet) green.

**Pull request merged**